doc/user_guide/user_guide.md

```diff
@@ -263,15 +265,57 @@ Please notice that we convert Avro complex types to the JSON Strings. Use Exasol
 `VARCHAR(n)` column type to store them. Depending on the size of complex type,
 set the number of characters in the VARCHAR type accordingly.
 
-## Import From Kafka Cluster
+## Importing Raw JSON
+
+When specifying ``RECORD_FORMAT=JSON`` the connector expects a valid UTF-8
+serialized JSON record per message. 
+When using ``ÀS_JSON_DOC=true``, the record is inserted as a whole and 
+the table has to be [prepared for it](#json-preparation)
+
+If you choose to import certain fields from the json record, specify the 
+``RECORD_FIELDS`` parameter with a comma separated list of fields to be imported
+
+``RECORD_FIELDS=age,lastName,address``
+
+and a json record like this
+
+```json
+  "firstName": "John",
+  "lastName": "Smith",
+  "isAlive": true,
+  "age": 27,
+  "address": {
+    "streetAddress": "21 2nd Street",
+    "city": "New York",
+    "state": "NY",
+    "postalCode": "10021-3100"
+  }
+```
+would allow you to import into a table with the following structure
+
+```sql
+CREATE OR REPLACE TABLE <schema_name>.<table_name> (
+    AGE         INTEGER,
+    LAST_NAME   VARCHAR(255),
+    ADDRESS     VARCHAR(10000),
+    -- Required for Kafka import UDF
+    KAFKA_PARTITION DECIMAL(18, 0),
+    KAFKA_OFFSET DECIMAL(36, 0),
+);
+```
+
+Note that the ``RECORD_FIELDS`` parameter is required when inserting JSON into
+columns as the order of fields in json records is not deterministic.
+
+# Import From Kafka Cluster
 
 Several property values are required to access the Kafka
 cluster when importing data from Kafka topics using the connector.
 
 You should provide these key-value parameters:
 
 - ``BOOTSTRAP_SERVERS``
-- ``SCHEMA_REGISTRY_URL``
+- ``SCHEMA_REGISTRY_URL`` (only when using avro)
```

Comment:
Let's remove it. It is already moved to optionals section.

Reply:
Okay, yes that makes sense.

Let us rearrange, I have added suggestions.

Reply:
Thanks for the suggestions. I reordered the points, fixed the table of contents plus added an example for importing JSON similar to the avro section